### PR TITLE
ovirt_vm: Check next_run configuration update if exist

### PR DESCRIPTION
**Pull request by machacekondra**
Backport of https://github.com/ansible/ansible/pull/47280

This PR fixes the update check method so it now check also the next_run
configuration of the virtual machine if it exists.

So if previously the VM was updated with new parameters, and then reset
back, the module didn't set the parameters to be set back in next_run.
This PR fixes it so the next run configuration is set back with proper
parameters.

Signed-off-by: Ondra Machacek <omachace@redhat.com>
Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=1639894
Signed-off-by: Ondra Machacek <omachace@redhat.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ANSIBLE VERSION

```paste below

```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
shipit

**Comment by mwperina** [comment]
Ryan, could you please merge? This is a customer related bug, so we would really need to get this into Ansible 2.7.1. Thanks!

@ryansb

**Comment by abadger** [comment]
Could you add a changelog entry for this?  It will be ready to merge once that is added.

**Comment by abadger** [comment]
@mwperina @machacekondra Also... today is the last day for merging for the 2.7.1 release.  So if you can get it in ASAP, that would be extremely helpful for getting it into 2.7.1 as opposed to 2.7.2.

**Comment by machacekondra** [comment]
@abadger Thanks for review. I've added the changelog.

**Review by mwperina — APPROVED** [review]
shipit

File: changelogs/fragments/ovirt_vm_check_next_run_configuration_update_if_exist.yaml
Comment (abadger) [inline changelogs/fragments/ovirt_vm_check_next_run_configuration_update_if_exist.yaml]: The problem in the changelog is the ": ".  In yaml, that has a special meaning, the separator between a dictionary's key and value elements.

The easiest fix is to change the ": " into a " - ".

**Review by mwperina — APPROVED** [review]
shipit

**Comment by abadger** [comment]
There are now conflicts in this PR (after merging other ovirt PRs)  If you have a chance to resolve those that would be great.  I'll take a look after I run through the backport queue at whether you've had a chance and whether it would be straightforward for me to resolve the conflict myself.

**Comment by abadger** [comment]
Okay, I'm looking at whether I can resolve the conflicts now.

**Comment by machacekondra** [comment]
@abadger conflicts resolved

**Comment by abadger** [comment]
@machacekondra Thanks.  github was still telling me there's conflicts, likely because of the merge (as opposed to rebase).  But I can see how to fix that thanks to your work.  I've pushed up a resolution to that.  Please review to be sure I haven't done anything wrong.

**Comment by machacekondra** [comment]
@abadger Change looks OK, thanks you!

**Comment by abadger** [comment]
Merged for the 2.7.1 release.

**Pull request merged**
